requestor-tutorials/service-development/service-example-2-managed-erigon.md

```diff
@@ -40,5 +40,221 @@ A typical interaction flow proceeds in the following manner:
 
 ## Requestor agent
 
+This part of the tutorial directly corresponds to the two previous requestor tutorials ([services hello world example](service-example-0-hello-world.md) and [simple service](service-example-1-simple-service.md)). We have the same clear separation between service specification and service provisioning, but there are important differences:
+
+* We use Erigon runtime instead of a VM-based runtime (so we don't have any Dockerfile or `image_hash`).
+* We don't implement `async def run` - the service is only started/stopped, requestor is idle when the service it is running.
+* We use [yapapi-service-manager](https://github.com/golemfactory/yapapi-service-manager) instead of pure `yapapi`.
+
+### Service specification
+
+As in previous tutorials, service specification is a subclass of `yapapi.Service`.
+Full code is available in the [yagna-erigon-repo](https://github.com/golemfactory/yagna-service-erigon/blob/master/requestor/server/erigon_service.py), here we'll discuss the important parts.
+
+#### Initialization:
+
+```python
+class Erigon(Service):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.url = None
+        self.auth = None
+        self.network = None
+```
+
+We add three attributes that are specific to our service logic:
+
+* `url` - where our provider will be accepting requests
+* `auth` - user/password pair that must be provided with each request
```

Comment:
These are related to Erigon RPC not the provider itself. 
Maybe replace `user/password` -> `credentials` (in my part there is `Erigon service's credentials` used)

Reply:
60b5f77